frontend/pages/policies/ManagePoliciesPage/components/PoliciesTable/PoliciesTable.tsx

```diff
@@ -55,34 +55,40 @@ const PoliciesTable = ({
 }: IPoliciesTableProps): JSX.Element => {
   const { config } = useContext(AppContext);
 
-  const emptyState = () => {
```

Comment:
There was no reason I could see for this to be a function that was called repeatedly to return the same results.  Unrolled to a standard var.

Reply:
🧼